Package.swift

```diff
@@ -22,19 +22,39 @@ let package = Package(
 	name: "Stagehand",
 	platforms: [
 		.iOS(.v12),
+		.macOS(.v11),
```

Comment:
Why is this necessary? A bit of a surprising change given that Stagehand does not support iOS (per the Podspec).

Reply:
It's required because of the snapshot-testing dependency requiring a newer macOS version than the SPM default (macOS 10.15). In SPM we can't actually say "we don't support this platform", we can only list which platform minimum we support, so this increases it to allow us to pull in swift-snapshot-testing

Reply:
Fascinating! Thanks for the explainer. I'd suggest adding a comment here so that this can be deleted in the future when minimum supported versions change (or when SPM gets better at this, which... hopefully sooner rather than later)